cpp/data/installer/scripts/install_centos_packages.sh

```diff
@@ -43,7 +43,7 @@ InstallIfMissing "ca-certificates"
 dnf --assumeyes install \
     ant bc cifs-utils clang crontabs ftp gcc-c++ glibc-all-langpacks git java-1.8.0-openjdk-devel make python3 sudo \
     gawk ImageMagick jq libcurl-devel libdb-devel libsq3-devel libuuid-devel libwebp libxml2-devel libxml2 lsof lz4 mariadb mariadb-devel mariadb-server mariadb-server-utils mod_ssl openssl-devel pcre-devel policycoreutils-python-utils poppler poppler-utils rpmdevtools unzip xerces-c-devel \
-    libcurl-devel tidy incron
```

Comment:
please restore alphabetical order
- libcurl-devel => duplicate
- tidy => should be between rpmdevtools & unzip
- incron => should be between ImageMagick & jq
- sqlite3 => should be between rpmdevtools & tidy